internal/match/regexp.go

```diff
@@ -22,9 +30,61 @@ package match
 //	expr??          - zero or one instances of expr
 //	//** text **//  - a comment
 //
+// To make patterns harder to misread in large texts:
+//
+//	- || must only appear inside (( ))
+//	- ?? must only follow (( ))
+//	- (( must be at the start of a line, preceded only by spaces
+//	- )) must be at the end of a line, followed only by spaces and ??.
+//
 // For example:
 //
 //	//** https://en.wikipedia.org/wiki/Filler_text **//
-//	Now is not?? the time for all good ((men || women))
+//	Now is
+//	((not))??
+//	the time for all good
+//	((men || women || people))
 //	to come to the aid of their __1__.
```

Comment:
Merge with comments in the parser.

Reply:
I can't tell what you mean by this.

Reply:
Figured it out, thanks for flagging. (GitHub really is awful for code reviews.)